src/util/isInteractiveElement.ts

```diff
@@ -0,0 +1,100 @@
+import { dom, elementRoles, roles } from 'aria-query';
+
+import { AXObjects, elementAXObjects } from 'axobject-query';
+
+import attributesComparator from './attributesComparator';
+import { ElementAst } from '@angular/compiler';
+
+const domKeys = [...(<string[]>Array.from(dom.keys()))];
```

Comment:
`(<string[]>Array.from(dom.keys()))`, this should be enough. No need to spread the array into another one